### DISCUSSION TITLE
java.lang.StackOverflow Error when downloading - Please help

### DISCUSSION THREAD

**Discussion by marcelomit** [ho-dev/HattrickOrganizer#1608] [General]
Hi, I a not a programmer, just a user (but IT professional)
I was using version 4.1 and start receiving this error and no data download occurs.
java.lang.StackOverflow Error
Upgraded to 5.2 and the same.
I have JDK 17 installed on windows 10.
I tried to uninstall all, including java, including deleting all files and cleaning register. The error persist even with portable jre versions.
How can I deal with this problem? what I am  missing? 
Thanks

**Comment by Asomers1204** [comment]
Same problem here in version 5.2 since last week. 
When downloading one of the new versions, I can't even open it anymore, get a Unsupported Class Version Runtime Error, while my Jave runtime is up-to-date
